src/sdam/description/topology/server_selection/test/in_window.rs

```diff
@@ -216,7 +217,7 @@ async fn load_balancing_test() {
     let mut handler = EventHandler::new();
     let mut subscriber = handler.subscribe();
     let mut options = CLIENT_OPTIONS.get().await.clone();
-    let max_pool_size = 10;
+    let max_pool_size = DEFAULT_MAX_POOL_SIZE;
```

Comment:
ditto here, I think we just want min/max pool size to be the same, so line 224 can just assign to `Some(DEFAULT_MAX_POOL_SIZE)`

Reply:
I removed `options.max_pool_size = Some(max_pool_size)`, but the `max_pool_size` variable is still needed nonetheless (it seems better to use it in multiple places in the method, rather than using `DEFAULT_MAX_POOL_SIZE` in those same places, and it is how it was done originally).